cluster-autoscaler/metrics/metrics.go

```diff
@@ -50,6 +50,12 @@ const (
 	LogLongDurationThreshold = 5 * time.Second
 )
 
+const (
+	ScaleDownNodeDeletion      = "scaleDown:nodeDeletion"
+	ScaleDownFindNodesToRemove = "scaleDown:findNodesToRemove"
+	ScaleDownMiscOperations    = "scaleDown:miscOperations"
```

Comment:
If you want to do it nicely maybe put all function names as consts here and give them a type (similar to NodeScaleDownReason above)? Though this is more of a drive-by cleanup and not really part of this PR, so don't feel obliged to do it.